docs/Doxyfile.in

```diff
@@ -23,7 +23,10 @@ MULTILINE_CPP_IS_BRIEF = YES
 
 TAB_SIZE               = 2
 
-ALIASES                = "semantics=<p></p><i>Semantics:</i>" \
+ALIASES                = "spectreversion=@SPECTRE_VERSION@" \
+                         "spectredoi=@SPECTRE_DOI@" \
```

Comment:
could add `spectrehompage` and `spectregitrepo` here too

Reply:
See https://github.com/sxs-collaboration/spectre/pull/2606#discussion_r533227169